gandi/cli/commands/domain.py

```diff
@@ -38,6 +38,18 @@ def info(gandi, resource):
 
 
 @cli.command()
+@click.argument('resource', metavar='DOMAIN', required=False)
```

Comment:
you must remove the required=False flag, as it is mandatory for your command to work